src/eventlog.h

```diff
@@ -195,6 +195,7 @@ enum {
  */
 typedef struct tpm_event_log_scan_ctx {
 	char *			efi_partition;
+	char *			prep_partition;
```

Comment:
Would make sense to have a new type that an  `union` of the efi or prep block? That way we can only have one or another, but not both

Reply:
It turned out that `prep_partition` in `tpm_event_log_scan_ctx` is just leftover code and not used anymore. On the other hand, `efi_gpt_event` also has `efi_partition` and `prep_partition`. I'll make them a union.

Reply:
After a further check for `efi_partition`/`prep_partition` in `efi_gpt_event`, those two can be merged into `sys_partition` since we only need to rehash the GPT table. Whether it is an EFI system partition or not, it doesn't matter.